src/app/app.module.ts

```diff
@@ -18,7 +19,7 @@ import { RecipeEditComponent } from './recipes/recipe-edit/recipe-edit.component
     RecipeComponent,
     RecipeEditComponent,
   ],
-  imports: [BrowserModule, RouterOutlet, AppRoutingModule],
+  imports: [BrowserModule, RouterOutlet, AppRoutingModule, ReactiveFormsModule],
   providers: [],
   bootstrap: [AppComponent],
 })
```

Comment:
This is bad practice, I know this project is small enough, but you shuould always encapsulate block of functionality within the modules. recipe should be a separate feature module, and app module should always import modules